### ISSUE TITLE
How to extract properties of nodes and add them in Neo4j?

### ISSUE THREAD

**Issue by ImsuperSH** [AuvaLab/itext2kg#23]
Does the project support it?Thanks

**Comment by lairgiyassir** [comment]
Hello, 

The node properties that are extracted and added in neo4j are : 
* Node label.
* Node name.
* Node embeddings. 

Do you want to add other properties?

**Comment by ImsuperSH** [comment]
> Hello,
> 
> The node properties that are extracted and added in neo4j are :
> 
> * Node label.
> * Node name.
> * Node embeddings.
> 
> Do you want to add other properties?

yes, I plan to extract some other properties from documents and add in neo4j. How to achieve it? Thanks.

**Comment by lairgiyassir** [comment]
Yes, it is possible. I have already added a data class called EntityProperty https://github.com/AuvaLab/itext2kg/blob/main/itext2kg/models/knowledge_graph.py

 However, you must fork this project and add the properties to the previous data model. Once done, all that you need to store the properties you would like to extract in these properties. 

For example 

```
class EntityProperties(BaseModel):
    embeddings: SkipValidation[np.array]=None
    prop1:SkipValidation[type of prop1]=None
    ...

    class Config:
        arbitrary_types_allowed = True
```
